### PR TITLE
Update last_fit to allow the passing of a control object

### LINKED ISSUE #399: control_last_fit()
The other `tune` functions have control arguments/functions. That's where we set options like `event_level`. 

See [this SO post](https://stackoverflow.com/questions/68561374/logistic-regression-with-tidymodels-how-to-set-event-level-second-in-last-fi/68616042).

### PR DESCRIPTION
**Pull request by mattwarkentin**
Fixes #399 (and related to #431)

### CODE REVIEW

File: R/control.R [hunk 1]
```diff
@@ -70,6 +70,41 @@ print.control_grid <- function(x, ...) {
...
+#'
+#' @details
+#'
+#' [control_last_fit()] is a wrapper around [control_resamples()] and is meant
+#'   to be used with [last_fit()].
+#'
+#' @export
+control_last_fit <- function(
+    verbose = FALSE,
+    pkgs = NULL,
```
Comment (juliasilge) [inline R/control.R:85]: Can you say more about why you included `pkgs` as an argument? Is this used for anything other than parallel processing? The other args that control parallel processing are not included, for good reason.

File: tests/testthat/_snaps/bayes.md [hunk 2]
```diff
@@ -57,38 +61,41 @@
...
 
 ---
 
     Code
       tune:::check_bayes_initial_size(5, 1, FALSE)
-    Error <rlang_error>
-      ! There are 5 tuning parameters and a single grid point was requested. The GP model requires 2+ initial points but there should be more initial points than there are tuning paramters. 
+    Condition
+      Error in `tune:::check_bayes_initial_size()`:
+      ! ! There are 5 tuning parameters and a single grid point was requested. The GP model requires 2+ initial points but there should be more initial points than there are tuning paramters.
```
Comment (juliasilge) [inline tests/testthat/_snaps/bayes.md:82]: Just want to highlight how the errors look now @topepo; we probably want to address this.

**Comment by juliasilge** [comment]
Looks like we may have a problem with oldrel-3 now, due to the new kernlab release?

File: R/control.R (hunk 1 above)
Comment (mattwarkentin) [reply R/control.R:85]: No, I think you are right. The `pkgs` argument can probably be removed. Want me to push this change?

Comment (mattwarkentin) [reply R/control.R]: Fixed with [cb41113](https://github.com/tidymodels/tune/pull/481/commits/cb41113a27ddccfa60890bc61a47e6b437a822b6)

File: tests/testthat/_snaps/bayes.md (hunk 2 above)
Comment (topepo) [reply tests/testthat/_snaps/bayes.md:82]: Do you mean the function called back? I'd leave as-is for this PR and we can add an issue to improve it overall.

**Comment by juliasilge** [comment]
> Looks like we may have a problem with oldrel-3 now, due to the new kernlab release?

We asked some experts and it looks like kernlab needs R 3.6.2, because they are missing `FCONE` on older versions. I took `oldrel-3` out, but we can likely add it back in after the next major version bump in R.

**Review by juliasilge — APPROVED** [review]
Thank you so much @mattwarkentin! 🙌

**Pull request merged**